### ISSUE TITLE
-vscode fix symbolic links to include directories

### ISSUE THREAD

**Issue by ghost** [KeyWorksRW/ttBld#163]
### Description:
Currently (8/2019), Intellisense gets confused if the path to additional include files is a symbolic link. What will happen is that the database won't get updated correctly. For example, assume a project **foo/src** with a symbolic link to **ttLib**.  The include directory is set to `../ttLib/Include`. Now change a header in the _real_ ttLib/Include and Intellisense won't be able to see the change.

Bottom line is that if the include directory is a symbolic link then we need to find the full path, and add that to c_cpp_properties.json.

### Issue Type:
- [ ] Bug report
- [x] Feature request
- [ ] Design change request
- [ ] Other
